### ISSUE TITLE
Homepage and download link gone, no releases in the repo

### ISSUE THREAD

**Issue by SanskritFritz** [octaspire/crates#4]
I hope you don't want to let this wonderful game die...

**Comment by octaspire** [comment]
Thank you for the kind words :-) and sorry about the missing downloads and releases for crates. It is nice to know that someone likes the game, and knowing that I could try to find some time to work with crates too.

Lately I have been interested in interpreters and compilers and the Lisp programming language. I have written a version of crates in my own Lisp dialect _octaspire dern_. The game can be seen in the YouTube video at the front page of [https://octaspire.io/](https://octaspire.io/) (starting at 2:22). It has some - possibly too many - differences with crates (2D vs 3D, missing enemy crates, faster movement, different "atmosphere" and so on) so that maybe it cannot take the place of crates.

**Comment by SanskritFritz** [comment]
I maintain an [Archlinux package](https://aur.archlinux.org/packages/crates/) for Crates and now I miss a release download. Thanks though for keeping the git repo here!

**Comment by SanskritFritz** [comment]
If you pick up development on Crates, I would love to see an option to turn off fullscreen mode and the possibility to set the resolution for fullscreen!
:smile:

**Comment by octaspire** [comment]
I tagged and created a release [v0.7.2](https://github.com/octaspire/crates/releases/tag/v0.7.2) for Crates. There are compressed archives available for that release.

I also fixed linking with math library in CmakeLists.txt.

I updated documentation about configuration options. It is already possible to change many configuration options in file **resources/config.lua**. User (or script) can toggle between fullscreen and windowed mode, change resolution or window size, toggle sound effetcs and music on/off, and so on. I tested this again, and it should work OK.
